app/controllers/api/base_controller/parameters.rb

```diff
@@ -1,6 +1,8 @@
 module Api
   class BaseController
     module Parameters
+      include CompressedIds
```

Comment:
@abellotti I *think* this can just be removed. Explanation here: https://github.com/ManageIQ/manageiq/pull/13671, tl;dr is these methods from this module are already available, you just can't reference any constants from the module unqualified from another module such as this.